docs/conceptual/using-gpu-sanitizer.md

```diff
@@ -102,6 +102,13 @@ This tells the ASan runtime to halt the application immediately after detecting
 
 This option directs the ASan runtime to enable the [Leak Sanitizer](https://clang.llvm.org/docs/LeakSanitizer.html) (LSan). Unfortunately, for heterogeneous applications, this default will result in significant output from the leak sanitizer when the application exits due to allocations made by the language runtime which are not considered to be leaks. This output can be avoided by adding `detect_leaks=0` to the `ASAN_OPTIONS`, or alternatively by producing an LSan suppression file (syntax described [here](https://github.com/google/sanitizers/wiki/AddressSanitizerLeakSanitizer)) and activating it with environment variable `LSAN_OPTIONS=suppressions=/path/to/suppression/file`. When using a suppression file, a suppression report is printed by default. The suppression report can be disabled by using the `LSAN_OPTIONS` flag `print_suppressions=0`.
 
+* `quarantine_size_mb=N default 256`
+
+This option defines the number of megabytes (MB) `N` of memory that the ASAN runtime will hold after it is `free`d to detect use-after-free situations.
```

Comment:
The ASAN occurrence here is all uppercase and not ASan

Reply:
I made them consistent with ASan